fdep/servers/xmlrpc.py

```diff
@@ -63,7 +65,7 @@ def register_functions(self, func_pairs):
     def serve_forever(self, **kwargs):
 
         if kwargs.get('username') and kwargs.get('password'):
-            XMLRPCRequestHandler.set_account(kwargs['username'], kwargs['password'])
+            XMLRPCRequestHandler.set_account(kwargs['username'], kwargs['password'], kwargs['temp_password'])
```

Comment:
I just want to point out that this will result in a key error if temp_password is not set.

Reply:
Might be safer to use `kwargs.get('temp_password')` and have logic to to handle the `None` case.

Reply:
Gotcha, for some reason I thought it would just evaluate to `None`. Will fix.